docker-compose.yml

```diff
@@ -1,8 +0,0 @@
-version: "3.6"
-services:
-  blazegraph:
-    build:
-      context: .
-      dockerfile: Dockerfile-blazegraph
```

Comment:
I think that may be someone forgot to commit that Docker-blazegraph and in general the idea to have it since IIRC blazegraph is used by the project for graphdb.  May be @albertcrowley  could clarify?